### ISSUE TITLE
Specify miniasm and minimap paths?

### ISSUE THREAD

**Issue by Kirk3gaard** [rrwick/Unicycler#119]
I think the post form @sjackman in #93 deserves an issue of its own:

"Is it possible to use minimap2 rather than minimap and miniasm -c2 rather than the default miniasm -c3 with Unicycler?"

We have indications that the latest version of minimap2 (2.11) results in more contiguous assemblies than the previous releases and it would be great to benefit from that in the future. Any chance that it would be possible to simply interface with these tools by specifying the path as is done for samtools, racon etc.?

**Comment by sjackman** [comment]
You can use the command line option `--existing_long_read_assembly` with a miniasm assembly that you assembled yourself.
See https://github.com/rrwick/Unicycler/releases/tag/v0.4.4
and https://github.com/rrwick/Unicycler/issues/93

**Comment by rrwick** [comment]
Yes, Shaun's suggestion is a good one. I didn't realise I'd left that out of the docs 😳 It's now in there!

A better solution would be for Unicycler to use minimap2 in the first place. But it's not a trivial switch, as Unicycler doesn't just call minimap, it's built into the C++ code. So I do hope to someday switch to minimap2, I just haven't gotten around to it yet!

Ryan
